packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -220,6 +222,11 @@ export const styles = (theme: ThemeType) => ({
   },
 });
 
+const cloudinaryBase = `${cloudinaryCloudNameSetting.get()}/image/upload/`;
+
+const formatImageUrl = (url: string) =>
+  url.replace(cloudinaryBase, `${cloudinaryBase}c_fill,w_124,h_70,dpr_2,`);
```

Comment:
This is a bit janky, but the URL is generated by some non-trivial logic that runs server-side that doesn't have anyway of configuring the parameters, and this doesn't feel like the right place to refactor that.